source/assets/public/index.css

```diff
@@ -54,6 +54,8 @@ body {
   text-rendering: optimizeLegibility;
   font-size: 1.8rem;
   line-height: 1;
+  color: var(--text-color);
+  font-family: var(--font-body);
```

Comment:
Setting these as defaults as eg `li` and `h3` items were coming up completely unstyled